app/views/customers/show.html.erb

```diff
@@ -15,10 +15,11 @@
 		</dd>
 
 	<dt>Requirements</dt>
-	<dd><%= @customer.require_background_checks? ? "Background checks<br>".html_safe : "" %>
-		<%= @customer.require_risk_form? ? "Mentor consent<br>".html_safe : "" %>
-		</dd>
-
+	<dd><%= @customer.require_background_checks? ? "Background checks<br>".html_safe : "Background Checks "+@customer.helper_background_check_validity_length+", "%>
```

Comment:
Same with lines 18 and 19, the way this is written, the validity length won't show unless the customer _doesn't_ require background checks.